Source/WebCore/platform/graphics/BitmapImageSource.h

```diff
@@ -162,7 +162,7 @@ class BitmapImageSource final : public ImageSource, public CanMakeCheckedPtr<Bit
     DestinationColorSpace colorSpace() const final { return m_descriptor.colorSpace(); }
     std::optional<Color> singlePixelSolidColor() const final { return m_descriptor.singlePixelSolidColor(); }
     bool hasHDRGainMap() const final { return m_descriptor.hasHDRGainMap(); }
-    bool hasHDRContent() const final { return m_descriptor.hasHDRGainMap() || m_descriptor.hasHDRColorSpace(); }
+    bool hasHDRContent() const final { return m_descriptor.hasHDRGainMap() || m_descriptor.hasHDRColorSpace() || hasHDRContentForTesting(); }
```

Comment:
Since you added `hasHDRContentForTesting()` to `BitmapImageSource::hasHDRContent()`. `BitmapImage::hasHDRContent()` does not have to check `hasHDRContentForTesting()` again.